lib/internal/errors.js

```diff
@@ -1159,6 +1159,7 @@ E('ERR_CHILD_PROCESS_STDIO_MAXBUFFER', '%s maxBuffer length exceeded',
   RangeError);
 E('ERR_CONSOLE_WRITABLE_STREAM',
   'Console expects a writable stream instance for %s', TypeError);
+E('ERR_CONSTRUCT_CALL_REQUIRED', '%s cannot call constructor without `new`', TypeError);
```

Comment:
```suggestion
E('ERR_CONSTRUCT_CALL_REQUIRED', 'Cannot call %s constructor without `new`', TypeError);
```
or, to mimic the error thrown by V8 when calling `(class Test{})()`
```suggestion
E('ERR_CONSTRUCT_CALL_REQUIRED', 'Class constructor %s cannot be invoked without `new`', TypeError);
```